etc/releasing.md

```diff
@@ -209,6 +208,49 @@ pushed.
   `docs/`. This may require pushing the commit to a fork of the C++ Driver
   repository and creating a pull request.
```

Comment:
I propose removing the "Push the commit containing changes to `etc/` and `docs/`." sentence entirely and rewriting this instruction to direct user to simply create a pull request from a fork repo (as done with normal feature branches) due to branch protections on `master` forbidding direct pushes.

The "create a PR with post-release changes" can also be postponed to include addition of `[Unreleased]` stubs in the CHANGELOG.